BreweryRatings/BreweryRatings.csproj

```diff
@@ -19,10 +19,6 @@
     </Content>
   </ItemGroup>
 
-  <ItemGroup>
-    <None Include="wwwroot\YelpAPIKey.txt" />
-  </ItemGroup>
```

Comment:
I imagine this is an external file that you want to have included in the project.